scripts/pre_commit_linter.py

```diff
@@ -160,6 +186,55 @@ def _lint_js_files(node_path, jscs_path, config_jscsrc, input_dir):
             num_js_files, time.time() - start_time)
 
 
+def _lint_py_files(config_pylint, input_dir):
+    """Prints a list of lint errors in changed Python files.
+
+    Args:
+    - config_pylint: str. Path to the .pylintrc file.
+    - input_dir: str. Path to the folder to be linted (optional).
+    """
+
+    print '----------------------------------------'
+
+    if input_dir:
```

Comment:
This repeats stuff. Can the list of files be generated in _pre_commit_linter() instead, and partitioned into python and JS files there? Then we just pass the relevant lists into these methods, which only focus on the actual linting.